aqua-csp-on-ubuntu-vm/add-acr-registries.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"vmName": {
+			"type": "string"
+		},
+		"addACRReg": {
+			"type": "string"
+		},
+		"aquaAdminPassword": {
+			"type": "securestring"
+		},
+		"azureADId": {
+			"type": "securestring"
+		},
+		"azureADPassword": {
+			"type": "securestring"
+		},
+		"azureTenantID": {
+			"type": "securestring"
+		}
+	},
+	"resources": [{
+		"type": "Microsoft.Compute/virtualMachines/extensions",
+		"name": "[concat(parameters('vmName'),'/installcustomscript')]",
+		"apiVersion": "2015-05-01-preview",
+		"location": "[resourceGroup().location]",
+		"properties": {
+			"publisher": "Microsoft.Azure.Extensions",
+			"type": "CustomScript",
+			"typeHandlerVersion": "2.0",
+			"autoUpgradeMinorVersion": true,
+			"settings": {
+				"fileUris": ["https://raw.githubusercontent.com/amitlib/azure-quickstart-templates/aqua-templates/aqua-csp-on-ubuntu-vm/addACRRegistry.sh"],
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts